projects/v3/src/app/services/experience.service.ts

```diff
@@ -248,6 +248,7 @@ export class ExperienceService {
 
     // initialise Pusher
     this.sharedService.initWebServices();
+    this.utils.checkIsPracteraSupportEmail();
```

Comment:
@sasangachathumal this type of initiator, it's recommended to move it to the `initWebServices()` as the name suggests, it's for initiations which is also suitable for the email support checking too.

Reply:
@trtshen Yes it's need moved. Will do that change.